docs/getting-involved-in-a-track.md

```diff
@@ -100,23 +100,23 @@ directory in the language track repository for the directory named after the
 slug of the exercise. The test file will be right there.
 
 That said, some test suites are generated based on the `canonical-data.json`
-file found in [x-common][]. The README for the language track repository
+file found in [x-common][https://github.com/exercism/x-common]. The README for the language track repository
```

Comment:
This fix breaks the link.

The `[x-common][]` is a shorthand for `[x-common][x-common]`, which references the `[x-common]: http://...` definition, which is defined at the top of the file.